### PR TITLE
 Added multi-language dropdown for course content.

### PR DESCRIPTION
**Pull request by komalahire**
1. Added dropdown for multi-language course content.
2. Have stored user-selected language option in localStorage.

### CODE REVIEW

File: src/components/Course/redux/api.js [hunk 1]
```diff
@@ -21,15 +21,15 @@ export const getCourses = () => {
...
  * @param {object} opts
  *
  * @returns {Promise}
  */
 export const getCourseContent = (data) => {
-  const { courseId } = data;
+  const { courseId, lang } = data;
   return axios({
-    url: `${process.env.REACT_APP_MERAKI_URL}/courses/${courseId}/exercises`,
+    url: `${process.env.REACT_APP_MERAKI_URL}/courses/${courseId}/exercises?userLang=${lang}`,
```
Comment (pysaquib) [inline src/components/Course/redux/api.js]: @komalahire Changed accepted query string to `lang` from the server please do the required changes.

File: src/pages/CourseContent/index.js [hunk 2]
```diff
@@ -49,8 +50,23 @@ function CourseContent(props) {
   const courseId = get(props, "match.params.courseId");
 
   useEffect(() => {
-    dispatch(courseActions.getCourseContent({ courseId: courseId }));
-  }, [dispatch, courseId]);
+    const getLocalStorageValue = localStorage.getItem("changeLanguage");
+    const valueSet =
+      getLocalStorageValue === undefined
+        ? setChangeLanguage("hi")
```
Comment (pysaquib) [inline src/pages/CourseContent/index.js]: Considering we cannot guarantee that every course will have Hindi content by default let's keep `en` as default option for now.

File: src/components/Course/redux/api.js (hunk 1 above)
Comment (pysaquib) [reply src/components/Course/redux/api.js]: @komalahire It is still userLang. Did you check the API in swagger?

**Review by pysaquib — CHANGES_REQUESTED** [review]

File: src/pages/CourseContent/index.js (hunk 2 above)
Comment (komalahire) [reply src/pages/CourseContent/index.js]: Done

File: src/components/Course/redux/api.js (hunk 1 above)
Comment (komalahire) [reply src/components/Course/redux/api.js]: Done

**Pull request merged**